templating/base_render_options.go

```diff
@@ -80,7 +86,11 @@ func LoadAndRenderSteps(template *Template, opts *BaseRenderOptions) (string, er
 	var err error
 
 	config := &Config{}
-	if opts.ValuesFile != "" {
+	if opts.Base64EncodedValuesFile != "" {
+		if config, err = DecodeConfig(opts.Base64EncodedValuesFile); err != nil {
+			return "", err
+		}
+	} else if opts.ValuesFile != "" {
```

Comment:
Base64EncodedValuesFiles precede ValuesFiles? Can you clarify it in readme or cmd options?

Reply:
done